ev3dev2/motor.py

```diff
@@ -1920,10 +2003,174 @@ def get_speed_steering(self, steering, speed):
             right_speed *= speed_factor
         else:
             left_speed *= speed_factor
-        
+
         return (left_speed, right_speed)
 
 
+class MoveDifferential(MoveTank):
+    """
+    MoveDifferential is a child of MoveTank that adds the following capabilities:
+
+    - drive in a straight line for a specified distance
+
+    - rotate in place in a circle (clockwise or counter clockwise) for a
+      specified number of degrees
+
+    - drive in an arc (clockwise or counter clockwise) of a specified radius
+      for a specified distance
+
+    New arguements:
+
+    wheel_class - A class typically from ev3dev2/wheel.py. This is used to
```

Comment:
I think we need a code example including imports at the bottom of this comment (see other classes for code formatting) to provide a copy-pastable example.